app/editor/src/features/admin/reports/utils/openPreviewInNewTab.ts

```diff
@@ -0,0 +1,19 @@
+import { IReportResultModel } from 'tno-core';
+
+export const openPreviewInNewTab = (previewContent: IReportResultModel) => {
+  const previewHTML = `
+      <html>
+      <head>
+        <title>Report Preview</title>
+      </head>
+      <body>
+        <h1> ${previewContent.subject}</h1>
+        <div>${previewContent.body}</div>
+      </body>
+      </html>
+    `;
+
```

Comment:
This is a good idea, but I think it needs to be fleshed out a bit with something other than just the subject and the body.  Once this new window or tab is opened, the context of what it is will be lost.  I would suggest a bit more markup at the top of the preview page that informs that it's a preview for a specific report, and also some sort of action (button) that can be taken to close the window/tab that is clearly tied to "Close Report Preview" or something.